postgres/client.sls

```diff
@@ -23,11 +23,14 @@ postgresql-client-libs:
     - refresh: True
     - require:
       - pkgrepo: postgresql-repo
+       {% if grains.os_family == 'Suse' %}
+      - cmd: postgresql-repo
+       {% endif %}
```

Comment:
I think here would be better to depend on the whole `postgres.upstream` SLS to reduce verbose Jinja statements:
```
    ...
    - require:
      - sls: postgres.upstream